supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -335,7 +335,7 @@ Write this name in full the first time that you use it in a document. Subsequent
 [discrete]
 [[runlevel]]
 ==== image:images/yes.png[yes] runlevel (noun)
-*Description*: A "runlevel" is a preset operating state on a Unix-like operating system. A system can be booted in to (that is, started up in to) any of several runlevels, each of which is represented by a single-digit integer. Each runlevel designates a different system configuration and allows access to a different combination of processes (that is, instances of executing programs). There are differences in the runlevels according to the operating system. Seven runlevels are supported in the standard Linux kernel.
+*Description*: A "runlevel" is a preset operating state on a UNIX system and similar operating systems. A system can be booted in to (that is, started up in to) any of several runlevels, each of which is represented by a single-digit integer. Each runlevel designates a different system configuration and allows access to a different combination of processes (that is, instances of executing programs). There are differences in the runlevels according to the operating system. Seven runlevels are supported in the standard Linux kernel.
```

Comment:
Fab improvements here @bredamc!

This comment is not part of your changes so feel free to ignore if it's beyond scope, but I wonder could we improve the second sentence, it seems unnecessarily clunky and troublesome for translation. Mainly because of the parentheses and the verb+preposition weirdness of ' started up in to'. Could probably avoid the word 'boot' altogether anyway. I don't think it'd change the meaning detrimentally to simplify it to something like:

`You can start a system into several runlevels, each of which is represented by a single-digit integer.` 

could replace the second set while we're there too:

`Each runlevel designates a different system configuration and allows access to a different combination of instances of executing programs.

Reply:
Thanks so much for your review, @ndeevy! I agree that the entire paragraph could use an edit, but I think it might be better to edit in a separate PR? I don't have enough expertise in this particular area so it would require significant review, which might delay the current PR. For example, is the system booted in several or one of several runlevels? Also, I think "processes" might be the better understood term, rather than "instances of executing programs". And is "allows" the correct verb here, or would "provides" be sufficient (not sure whether permissions are involved)? So many questions :)